share/spack/gitlab/nersc_pipeline.yml

```diff
@@ -0,0 +1,9 @@
+merge_pipeline:
+  only:
+  - develop
+  variables:
+    SPACK_REPO: ${CI_PROJECT_URL}
+    SPACK_REF: ${CI_COMMIT_SHA}
+  trigger:
+    project: ecp/e4s/e4spipelines/e4s-luke
```

Comment:
I haven't verified that this is the right project path.  Seems like a lot of slashes, but maybe that's how it's set up?  Out of curiosity, is the organization `ecp` and the rest is the project?

Reply:
@scottwittenburg : I checked the path and file with my fork and it is working as expected. ECP and E4S are groups everything else is project. We are hoping to keep ECP teams within some sort of structure